lms/djangoapps/teams/static/teams/js/views/edit_team.js

```diff
@@ -6,15 +6,16 @@
             'gettext',
             'js/views/fields',
             'teams/js/models/team',
+            'common/js/components/utils/view_utils',
             'text!teams/templates/edit-team.underscore'],
-        function (Backbone, _, gettext, FieldViews, TeamModel, editTeamTemplate) {
+        function (Backbone, _, gettext, FieldViews, TeamModel, ViewUtils, editTeamTemplate) {
             return Backbone.View.extend({
 
                 maxTeamNameLength: 255,
                 maxTeamDescriptionLength: 300,
 
                 events: {
-                    'click .action-primary': 'createOrUpdateTeam',
+                    'click .action-primary': function(event) {this.disableRun(event, this.createOrUpdateTeam);},
```

Comment:
It "works", to the point that it tries to call this, but fails with `operation is not a function` inside of disableElementWhileRunning.

I had to wrap everything here in an anonymous function due to `this.createOrUpdateTeam` not being defined at the time this event map is declared.